src/commands/roll.ts

```diff
@@ -6,7 +6,7 @@ export const description = 'feeling lucky? use this command to roll a N sided di
 export default ({ channel, context, args }: ICommandProps) => {
     if (args.length && !Number.isNaN(args[0])) {
         const max = Math.floor(Number(args[0]))
-        if (max > 0) {
```

Comment:
change this back, you can't roll 0 on a die

Reply:
got it